src/configure/resources/tracePoints.ts

```diff
@@ -0,0 +1,34 @@
+export class TracePoints {
+    // Combined telemetry trace points
+    public static telemetryKey: string = '';
+    public static CurrentFunction: string = 'currentFunction';
+    public static RepoProvider: string = 'repoProvider';
+    public static AzureLoginRequired: string = 'azureLoginRequired';
+    public static Command: string = 'command';
+    public static JourneyId: string = 'journeyId';
+    public static EntryPoint: string = 'entryPoint';
+    public static SourceRepoLocation: string = 'sourceRepoLocation';
+    public static NewOrganization: string = 'newOrganization';
+    public static ChosenTemplate: string = 'chosenTemplate';
+    public static PipelineDiscarded: string = 'pipelineDiscarded';
+    public static ViewPipelineClicked: string = 'viewPipelineClicked';
+    public static GitHubPatStartTime = 'gitHubPatStartTime';
+    public static GitHubPatEndTime = 'gitHubPatEndTime';
```

Comment:
Is this interesting?

Reply:
This was an ask from pulkit. This is basically a data point that can help us in prioritizing the GitHub OAuth flow.